apps/vmq_server/test/vmq_publish_SUITE.erl

```diff
@@ -459,6 +460,87 @@ publish_c2b_retry_qos2_test(_Config) ->
     ok = expect_alive(Socket),
     ok = gen_tcp:close(Socket).
 
+publish_b2c_ensure_valid_msg_ids_test(Config) ->
```

Comment:
it would be interesting to test what happens if multiple of such PUBRELs were persisted.

Reply:
Which test scenario are you thinking about here? I agree more cases could/should be covered, ideally we should property test stuff like this, if possible.